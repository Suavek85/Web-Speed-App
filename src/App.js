import React from 'react'
import InputCard from '../src/components/Input/InputCard/InputCard'
import Results from '../src/components/Results/Results'
import Navigation from '../src/components/Navigation/Navigation'
import './App.scss'

function App() {
  return (
    <div className='app__wrapper'>
      <Navigation />
      <InputCard />
      <Results />
    </div>
  )
}

export default App
