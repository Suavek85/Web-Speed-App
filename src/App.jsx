import React from 'react'
import { Switch, Route } from "react-router-dom"
  
import Homepage from './components/Homepage/Homepage'
import SigninPage from './components/SignIn/SigninPage'

export default function App() {

  return (
    <>
      <Switch >
        <Route exact path="/" component={ Homepage } />
        <Route exact path="/signin" component={ SigninPage } />
      </Switch>
    </>
  )

}

