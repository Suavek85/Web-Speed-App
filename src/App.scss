.app__wrapper {
    display: flex;
    position: relative;
    background-color: #A8D0E6;
}

:export {
    block: app__wrapper;
}