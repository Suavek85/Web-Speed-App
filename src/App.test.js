import React from 'react'
import { shallow } from 'enzyme'
import App from './App'

it('should match the snapshot', () => {
  const wrapper = shallow(<App />)
  expect(wrapper).toMatchSnapshot()
})