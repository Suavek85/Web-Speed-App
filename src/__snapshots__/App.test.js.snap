// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<Fragment>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/signin"
    />
  </Switch>
</Fragment>
`;
